### ISSUE TITLE
Track forbidden/permission granted for major brands in data file

### ISSUE THREAD

**Issue by uncenter** [simple-icons/simple-icons#11270]
### Discussed in https://github.com/simple-icons/simple-icons/discussions/11269

<div type='discussions-op-text'>

<sup>Originally posted by **jrhaberland** June 18, 2024</sup>
I'm pretty sure this doesn't exist in the repository as I've looked pretty thoroughly for it. I feel like there should be some place that keeps a log of the companies that have given simple-icons permission to use their icons.

If it were to be made it should store the following:
- Name of company
- Date permission was granted
- A copy of the email that gave said permission
- Link to related issues and or PRs
- Maybe a list of all the icons related with the company</div>

**Comment by uncenter** [comment]
Kinda piggybacking off of #9999, a `_data/permissions.json` file or similar would be nice to track explicitly forbidden brands or places were permission has been granted. I'm thinking something like the following:
```jsonc
[
  {
    "brand": "General Motors",
    "status": "approved", // or "denied" or "forbidden"
    "related": [8837, 3984], // list of related issues/PRs
    "context": [
      { type: "comment", content: "https://github.com/simple-icons/simple-icons/pull/11150#issuecomment-2174356643" }
    ],
  },
  // ...
]
```

**Comment by jrhaberland** [comment]
Maybe could add functionality to take and use that to create a regex that can be used with the auto closer bot github action.

**Comment by uncenter** [comment]
Yep that's the idea of https://github.com/simple-icons/simple-icons/issues/9999.

**Comment by jrhaberland** [comment]
This could also be a good way to test one of the different ways of changing the scheming of the json files as discussed in #9368

**Comment by adamrusted** [comment]
Returning to this, given there are new brands that would benefit from this being pushed. I think we should also track the 'slug' in this file - so that we can better match this list against the icons JSON for those that have been approved. In your 'status' field, I would also add 'implicit' for icons that we do not have evidence of approval for, but are otherwise okay through licenses etc.

Perhaps we start showing an explicit/implicit approval for use icon on the website? As demonstrated by #12378 we're likely to have our catalog decimated by not having proper permissions, so a system to track them would be great.
